### ISSUE TITLE
Unable to install - drag-and-drop method

### ISSUE THREAD

**Issue by DepressedOptimist** [TheFantasticWarrior/chrome-extension-imagus#47]
Hi,
I just tried to install using drag-and-drop onto the Extensions page and got the following error; developper mode is enabled.

```
Error Loading Extension
Failed to load extension from: D:\chrome-extension-imagus-new.zip.
Could not unzip extension for install.
```

Chrome Version 117.0.5938.89 (Official Build) (64-bit)

**Comment by kuzn123** [comment]
See [FAQ](http://forum.ru-board.com/topic.cgi?forum=5&topic=50874&start=0&limit=1&m=7#1), p.23 - "How to install Imagus Mod in Chrome/Edge".

**Comment by DepressedOptimist** [comment]
Thank you. I followed the instructions for Chromium when installing on both Edge and Chrome. The extension was disabled by default on both, but I found [this reference](https://techjourney.net/chrome-edge-disables-crx-installed-extensions-workarounds-to-turn-on/) online that shows a modification to the registry that will allow the extension to work.

EDGE
The registry, under HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\ did not have Edge. So, I created the Edge key, then under Edge, I created the key and string value the post mentions.

CHROME
The registry, under HKEY_LOCAL_MACHINE\SOFTWARE\Policies\ did not have Google at all, so I created the Google\Chrome keys, then added the key and string value the post mentions. IT DID NOT WORK. The post says to create the key 'ExtensionInstallWhitelist' for Chrome, but since this post was from 2020, I thought Chrome updated the key name, so I tried 'ExtensionInstallAllowlist' instead and it worked.

NOTE TO ANYONE READING THESE INSTRUCTIONS
Messing around in the registry can create system instability. Do not do this if you don't know what you are doing.

The article states that this only works on Windows Enterprise, however I'm on Windows 11 PRO and it worked.
